### PR TITLE
Dmceachernmsft/meetingParticipantNamesAvatarsSidePane

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
changed people pane in meeting composite to use the participantContainer component.

# Why
Allows for Avatar Customization in the people pane in the side bar. 

https://skype.visualstudio.com/SPOOL/_workitems/edit/2694576
# How Tested
Ran locally in meeting sample to see if people pane changed avatar information.

### CODE REVIEW

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
 // Licensed under the MIT license.
 import React from 'react';
-import { participantListStack, participantListStyle, participantListWrapper, listHeader } from './styles/Chat.styles';
-import { OnRenderAvatarCallback, ParticipantList, ParticipantMenuItemsCallback } from '@internal/react-components';
+import {
+  participantListStack,
+  participantListStyle,
+  participantListWrapper,
+  listHeader
+} from '../ChatComposite/styles/Chat.styles';
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/common/ParticipantContainer.tsx:9]: These styles should also move to `common`. Stuff from `common` shouldn't import from `ChatComposite`.

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
+  OnRenderAvatarCallback,
+  ParticipantList,
+  ParticipantListProps,
+  ParticipantMenuItemsCallback
+} from '@internal/react-components';
 import { FocusZone, Stack } from '@fluentui/react';
 import { useLocale } from '../localization';
-import { usePropsFor } from './hooks/usePropsFor';
-import { AvatarPersona, AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/common/ParticipantContainer.tsx:18]: This is a bit of a red flag, that code from `common` is calling `ChatComposite/hooks/usePropsFor`.

`usePropsFor` from there might only work for chat components.  Looking at how this is used below, I'd recommend that `ParticipantContainerProps.participatnListProps` be a required field, that is populated with `usePropsFor` from inside `ChatComposite` and `MeetingComposite`.

**Review by prprabhu-ms — COMMENTED** [review]
Will need changefiles.

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx (hunk 2 above)
Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:18]: okay I will clean this up!

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
   ParticipantList,
   ParticipantListProps,
   ParticipantMenuItemsCallback
 } from '@internal/react-components';
-import { FocusZone, Stack } from '@fluentui/react';
-import { useLocale } from '../localization';
-import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
+import { concatStyleSets, FocusZone, Stack, useTheme } from '@fluentui/react';
 import { AvatarPersona, AvatarPersonaDataCallback } from './AvatarPersona';
+import { peopleSubheadingStyle } from './styles/SidePane.styles';
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/common/ParticipantContainer.tsx:13]: naming nit: `SidePange.styles` harkens to how this component is used by the `MeetingComposite` (but not `ChatComposite`. How about `ParticipantContainer.styles` :)

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
  * @private
  */
 export const ParticipantContainer = (props: ParticipantContainerProps): JSX.Element => {
-  const { onFetchAvatarPersonaData, onFetchParticipantMenuItems } = props;
-  const participantListProps = props.participantListProps ? props.participantListProps : usePropsFor(ParticipantList);
-  const locale = useLocale();
-  const chatListHeader = locale.strings.chat.chatListHeader;
+  const { onFetchAvatarPersonaData, onFetchParticipantMenuItems, title, participantListProps } = props;
+  const theme = useTheme();
+  const subheadingStyleThemed = concatStyleSets(peopleSubheadingStyle, {
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/common/ParticipantContainer.tsx:29]: Not a 100% sure why this was needed.

**Comment by prprabhu-ms** [comment]
Would like to see snapshot updates (if any) before approval.

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx (hunk 3 above)
Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:13]: makes sense to me! will take care of this

File: packages/react-composites/src/composites/common/ParticipantContainer.tsx (hunk 4 above)
Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:29]: the styling? just to use the styles that were there before, I am not 100% sure they worked correctly so I am waiting on the diffs to see

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/common/ParticipantContainer.tsx:29]: > Would like to see snapshot updates (if any) before approval.

they updated the snapshots and it appears that there is a scroll bar present on the small version, something that I think is a symptom of the ParticipantContainer wrapping the ParticipantList.

**Review by prprabhu-ms — COMMENTED** [review]
image

The shadow doesn't look right.

**Review by prprabhu-ms — COMMENTED** [review]
image

As you noted, the scrollbar ain't wanted either.

**Comment by dmceachernmsft** [comment]
> image
> 
> The shadow doesn't look right.

I think its a symptom of using the ParticipantContainer, I'll look into it!

File: change/@internal-react-composites-5523c033-6f94-4a66-9fb6-5910d071fb8c.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (JamesBurnside) [inline change/@internal-react-composites-5523c033-6f94-4a66-9fb6-5910d071fb8c.json:2]: nit: going by the comment `"changed meeting peopel pane to use ParticipantContainer Component."` this should be type: none as this is just an internal refactor (so no implication on customers).

**Review by JamesBurnside — APPROVED** [review]

**Review by prprabhu-ms — APPROVED** [review]

**Review by PorterNan — APPROVED** [review]
Let's make a separated PR to create a single file and do all the conditional stuff there, then import it in the file (so that we can minimized /* conditional-xxx */ comments)

**Pull request merged**